src/base/private/profile_p.h

```diff
@@ -48,7 +48,12 @@ namespace Private
 
         virtual ~Profile() = default;
 
-        QString configurationName() const;
+		/**
+		 * @brief Configuration name suitable to use as directory name
+		 *
+		 * @return QCoreApplication::applicationName() with optional configuration name appended
+		 */
+		QString configurationName() const;
```

Comment:
The name of this function is misleading (see comment above and `Private::Profile::Profile(const QString &configurationName)` implementation. Could anyone suggest a better one, please?